minecraft_servers/blotcraft/manifest.json

```diff
@@ -0,0 +1,71 @@
+{
+	"server_name": "blotcraft",
+	"nice_name": "Blotcraft",
+	"direct_ip": "blotcraft.com",
+	"server_wildcards": [
+		"blotcraft.%"
+	],
+	"supported_languages": [
+		"en",
+		"es"
+	],
+	"social": {
+		"web": "https://web.blotcraft.com/",
+		"web_shop": "https://blotcraft.shop",
+		"web_support": "https://web.blotcraft.com/foro",
+		"twitter": "BlotCraft",
+		"tiktok": "BløtCraft",
+		"facebook": "BlotCraft",
+		"instagram": "blotcraft",
+		"discord": "https://discord.blotcraft.com",
```

Comment:
"Own urls" are only supported for partners. Please use the official discord invite link.

Documentation reference: https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url